apps/daimo-mobile/src/sync/recipients.ts

```diff
@@ -10,6 +10,18 @@ export interface Recipient extends EAccount {
   lastSendTime?: number;
 }
 
+/** Adds lastSendTime to an EAccount using account's recentTransfers */
+export function extendToRecipient(
```

Comment:
i'd call this `addLastSendTime` , current name it's hard to tell what it does

Reply:
done